rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -98,6 +99,52 @@ def put(self, deployment_id, **kwargs):
         )
         return deployment, 201
 
+    @authorize('deployment_create')
```

Comment:
This should perhaps be a different permission- but maybe we should wait to review permissions at some point?

Reply:
That's a good question. Since labels are always in the context of an object, I think their creation permissions should be the same as the object's.